docs/docs/topics/load-balancing.md

```diff
@@ -43,18 +45,31 @@ policy:
         http_health_check:
           path: "/"
 ```
+
 ### Passive Health Checks
 
 Passive health check tries to deduce upstream server health based on recent observed responses.
-See [Outlier Detection](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/outlier) for a comprehensive overview.
+See [Outlier Detection](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/outlier) for comprehensive overview.
+
+```yaml
+policy:
+  - from: https://myapp.localhost.pomerium.io
+    to:
+      - http://myapp-srv-1:8080
+      - http://myapp-srv-2:8080
+    outlier_detection: {}
+```
 
 ## Load Balancing Method
 
-`lb_policy` should be set to one of the values:
+`lb_policy` should be set to [one of the values](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/load_balancers):
```

Comment:
Should we use the version pinned version of envoys docs?

Reply:
my understanding is that Envoy's V3 API/Configs are frozen (they have a lot of currently hidden, not implemented features).